src/data/faq_de.json

```diff
@@ -87,6 +87,126 @@
+                        "anchor": "rat_privileges",
+                        "active": true,
+                        "textblock": [
+                            "Ein negatives Testergebnis kann Ihnen, falls gesetzlich festgelegt, als Nachweis für das Vorliegen eines negativen Schnelltest-Ergebnisses dienen. Die Anerkennung von Test-Nachweisen kann allerdings von Bundesland zu Bundesland variieren. Sie sollten sich deshalb über die jeweiligen Kriterien in Ihrem Bundesland informieren."
+                        ]
+                    },
+                    {
+                        "title": "Wie handele ich, wenn mein Schnelltest-Ergebnis positiv ist?",
+                        "anchor": "rat_what_if_positive_result",
+                        "active": true,
+                        "textblock": [
+                            "In Fall eines positiven Testergebnisses sollten Sie Ihr Testergebnis teilen und umgehend einen PCR-Test machen, um das Ergebnis zu bestätigen oder zu widerlegen. Bei einem positiven PCR-Test sollten Sie das Testergebnis erneut teilen, um Kontakte zu warnen, die Sie möglicherweise zwischen den beiden Tests getroffen haben. Voraussetzung dabei ist, dass die Kontaktpersonen ebenfalls die Corona-Warn-App nutzen und die Risikoberechnung ein hohes Risiko bestimmt, basierend auf Dauer und Abstand des Kontakts. Sollte der PCR-Test negativ sein, haben Sie nicht die Möglichkeit, Ihren positiven Schnelltest und die Warnung zurückzuziehen."
+                        ]
+                    },
+                    {
+                        "title": "Wenn ich eine hohe Risikobegegnung in der App angezeigt bekomme, d.h. eine rote Karte/Kachel, sollte ich dann einen Schnelltest oder PCR-Test machen?",
+                        "anchor": "rat_red_card_which_test",
+                        "active": true,
+                        "textblock": [
+                            "Wenn die Corona-Warn-App Ihnen eine Begegnung mit erhöhtem Risiko (rote Kachel) anzeigt, sollten Sie einen PCR-Test machen und sich in Quarantäne begeben."
+                        ]
+                    },
+                    {
+                        "title": "Werden Schnelltest und PCR-Test in der App unterschiedlich behandelt?",
+                        "anchor": "rat_same_in_cwa",
+                        "active": true,
+                        "textblock": [
+                            "Der Schnelltest wird von der Corona-Warn-App gleichberechtigt zum PCR-Test behandelt. Wenn Sie sich bei einem Corona-Warn-App Partner testen lassen, erhalten Sie Ihr Testergebnis per QR-Code, den sie mit der Corona-Warn-App scannen können. Ein negatives Testergebnis wird dann für 48 Stunden in der Corona-Warn-App angezeigt.",

```

Comment:
Corona-Warn-App-Partner
Oder besser: Bei einem Partner der Corona-Warn-App testen lassen,...

Reply:
If this get changed English needs to be aligned.

Reply:
ok